docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -342,14 +295,13 @@ you can only test via HTTP calls. Your test code does not actually run natively,
 that does not call your HTTP endpoints, it's probably not a good idea to run them as part of native tests.
 
 If you share your test class between JVM and native executions like we advise above, you can mark certain tests
-with the `@DisabledOnNativeImage` annotation in order to only run them on the JVM.
+with the `@DisabledOnIntegrationTest` annotation in order to only run them on the JVM.
```

Comment:
To be honest I don't know if `@DisabledOnNativeImage` would even work in a test class annotated with `@QuarkusIntegrationTest`?

Reply:
My point is more that DisabledOnIntegrationTest applies across the board.. not just for native images.

Reply:
It does not

Reply:
Maybe the correct language should be

`If you share your test class between JVM and integration test executions like we advise above, you can mark certain tests with the @DisabledOnIntegrationTest annotation in order to only run them on the JVM`.

?

Reply:
@ebullient / @geoand how does this sound?

`If you share your test class between JVM and native executions like we advise above, you can mark certain tests
with the `@DisabledOnIntegrationTest` annotation in order to skip them when testing against a native image.`

Reply:
Makes sense

Reply:
I went with this & pushed a new commit.

Reply:
I still don't think that is right, is it? Won't using that annotation ALSO disable the tests in JVM mode (as that is just the integration test phase?)

Reply:
But we're talking about building a native image (its the native image guide). It will disable the test when integration tests are run (i.e. maven failsafe).

The text as I wrote is accurate. Any test annotated with `@DisabledOnIntegrationTest` will not be run when building/running integration tests against the native image.

Reply:
I added a note underneath as well...

```
NOTE: Using @DisabledOnIntegrationTest will also disable the test in all integration test instances, including
testing the application in JVM mode, in a container image, and native image.
```